v2/community/reusableMD/supported-tech-stacks-frontend-custom-ui.mdx

```diff
@@ -5,12 +5,12 @@
   justifyContent: "start",
   flexWrap: "wrap"
 }}>
-<img src="/img/logos/react.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/vue.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/angular.svg" style={{width: "65px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/android.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/ios.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/electron.svg" style={{width: "60px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/capacitor-js.svg" style={{width: "60px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
-<img src="/img/logos/js_logo.svg" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
+<img src="/img/logos/react.svg" alt="react logo" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
```

Comment:
```suggestion
<img src="/img/logos/react.svg" alt="React logo" style={{width: "50px", margin: "0px", marginRight: "40px", marginTop: "10px"}}/>
```
Make similar changes for other logos in the file as well

Reply:
done